sections/semantics-sections.include

```diff
@@ -359,16 +368,18 @@
   </p>
 
   <p class="note">
-    Authors are encouraged to use the <{article}> element instead of the
-    <{section}> element when the content is complete, or self-contained, composition.
+    Unlike an <{article}>, a <{section}> may not be fully understandable if taken out of context
+    of its originating document or application. Authors are encouraged to use the <{article}>
+    element instead of the <{section}> element when the content is complete, or self-contained,
```

Comment:
Add "a" into "the content is *a complete, or self-contained..."

Reply:
Thanks @LJWatson!  updated and will merge after checks finish. :)